### ISSUE TITLE
Fix S1075: should not report on virtual path on asp.net bundles

### ISSUE THREAD

**Issue by andrei-epure-sonarsource** [SonarSource/sonar-dotnet#2357]
### Description

S1075 should not raise for virtual paths given to ASP.NET MVC **Bundle** classes, as it's a common pattern to pass hardcoded virtual paths. We already have a list of ASP.NET exceptions to the rule ([see description](https://rules.sonarsource.com/csharp/RSPEC-1075)), we need to add exceptions for classes inheriting from `ScriptBundle` and `StyleBundle` (from `System.Web.Optimization`).

Issue reported via [sonarsource community forum](https://community.sonarsource.com/t/false-positive-on-s1075-in-asp-net-mvc-with-paths/843/3)

```
public static void RegisterBundles(BundleCollection bundles)
{
     bundles.Add(new ScriptBundle("~/bundles/jquery") // should not raise
       .Include("~/Scripts/jquery-{version}.js")); // should not raise
}
```

### Related information

* SonarC# Version  - 7.11

Related tickets: #1865 #1586

**Comment by dmitrypodrezoff** [comment]
Also had this warning for `MvcAreaRouteBuilderExtensions.MvcAreaRouteBuilderExtensions` and `HealthCheckApplicationBuilderExtensions.UseHealthChecks` extension methods in Startup.
